_data/accessibility-tests/memorable-date.yml

```diff
@@ -0,0 +1,79 @@
+title: Memorable date
+component_status: pass
+test_items:
+# General tests
+  - summary: Memorable date fields announce their purpose.
+    summary_additional: When you navigate to the select and input fields, you hear that the input expects the day, month, or year.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.5
+  - summary: Touch target size for all clickable elements meets height and width requirements.
+    summary_additional: The size and spacing of the touch target are at least 24px wide and 24px tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.5.8
+  - summary: Element label text and background meet color contrast requirements.
+    summary_additional: When you use ANDI or another color contrast analyzer on memorable date, all labels and text inside elements meet a 4.5:1 WCAG contrast ratio.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
```

Comment:
**issue:** It looks like all of these criteria were retested on 8/15/24, which is when 3.8.2 was out. 

```suggestion 
    version_tested: 3.8.2
```

Reply:
Resolved in [a23850d](https://github.com/uswds/uswds-site/pull/2919/commits/a23850dd15036feb42d10dcb9257ba2f83e2ad14)